opportunity_job_app/src/api/models/User.ts

```diff
@@ -2,4 +2,5 @@ export interface User {
     token: string;
     id: number;
     accountType: string;
+    account_id: number;
```

Comment:
Ovde bih koristio `snake_case` za sva polja u modelu jer bi trebalo da polja budu istao kao i u modelu na BE, tj. kao sto response vraca sa BE.
Drugim recima, promenio bih `accountType` u `account_type`.